spec/models/form526_submission_remediation_spec.rb

```diff
@@ -39,21 +39,28 @@
         subject.mark_as_unsuccessful(remediate_context)
       end
 
-      it 'is invalid if ignored_as_duplicate is true and success is false' do
-        subject.ignored_as_duplicate = true
+      it 'is invalid if remediation_type is ignored_as_duplicate and success is false' do
+        subject.remediation_type = :ignored_as_duplicate
         subject.success = false
         expect(subject).not_to be_valid
       end
 
       it 'is valid if ignored_as_duplicate is true and success is true' do
-        subject.ignored_as_duplicate = true
+        subject.remediation_type = :ignored_as_duplicate
         subject.success = true
         expect(subject).to be_valid
       end
     end
   end
 
   describe 'instance methods' do
+    describe 'ignored_as_duplicate?' do
```

Comment:
Although this method is within an 'instance methods' block (which is probably an unnecessary addition when I drafted this spec), I think it's common convention to include a # before the method name.